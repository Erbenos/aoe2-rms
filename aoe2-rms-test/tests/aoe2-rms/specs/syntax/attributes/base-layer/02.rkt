#lang aoe2-rms
; Create one hill on water terrain.

<LAND-GENERATION>
(base-terrain 'DIRT)
(base-layer 'SNOW)

<ELEVATION-GENERATION>
(create-elevation 7
                  (base-terrain 'DIRT3)
                  (base-layer 'SNOW)
                  (number-of-tiles 9320)
                  (number-of-clumps 20))
