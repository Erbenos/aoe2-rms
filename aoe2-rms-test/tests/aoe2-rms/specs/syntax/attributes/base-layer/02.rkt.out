<LAND_GENERATION>
base_terrain DIRT
base_layer SNOW
<ELEVATION_GENERATION>
create_elevation 7 {
  base_terrain DIRT3
  base_layer SNOW
  number_of_tiles 9320
  number_of_clumps 20
}
